src/nplinker/data/nplinker.toml

```diff
@@ -1,321 +1,63 @@
+#############################
 # NPLinker configuration file
-# ---------------------------
+#############################
+
+# The root directory of the NPLinker project. You need to create it first.
+# The value is required and must be a full path.
+root_dir = "<NPLinker root directory>"
```

Comment:
Will the user need to edit the toml file manually for setting up this or is there going to be a way to do that from the code API?

Reply:
To run nplinker, users will first provide a config file. If they want to change the settings, it's possible to directly update the value, e.g. `config.set("root_dir", "new_path")`.